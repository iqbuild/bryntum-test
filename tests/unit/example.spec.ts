import { shallowMount } from '@vue/test-utils';
import HelloWorld from '@/components/Example/Example.vue';

describe('Example.vue', () => {
	it('renders', () => {
		const msg = 'Example';
		const wrapper = shallowMount(HelloWorld, {
			props: { msg },
		});
		expect(wrapper.text()).toMatch(msg);
		expect(wrapper.element).toMatchSnapshot();
	});
});
