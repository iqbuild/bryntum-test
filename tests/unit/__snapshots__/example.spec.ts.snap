// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example.vue renders 1`] = `
<div
  msg="Example"
>
  Example
</div>
`;
